src/test/ui/uninhabited/exhaustive-wo-nevertype-issue-51221.rs

```diff
@@ -0,0 +1,9 @@
+// check-pass
```

Comment:
Seems the filename has a typo? "w" instead of "n"

Reply:
wo = without 😅
I was not sure how to name it. I didn't want only "issue-XXX" but also in this case describing that this is a general problem. 
Using exhaustive patterns should be possible regardless of the issue number.

Reply:
Oh, I see; I thought you intended `no-nevertype` ^^